appveyor.yml

```
@@ -39,7 +39,9 @@ for:
     build_script:
       - make -C Build -j install
       - pkg-config --debug --exists --print-errors SvtVp9Enc
-      - git clone https://github.com/FFmpeg/FFmpeg.git --depth=1 --branch=master ffmpeg
+      - ps: Invoke-WebRequest -UseBasicParsing -OutFile ffmpeg.zip -Uri "https://github.com/FFmpeg/FFmpeg/archive/release/4.2.zip"
+      - 7z x -aoa -offmpeg ffmpeg.zip
+      - ps: Get-ChildItem ffmpeg/FFmpeg-*/* | ForEach-Object {Move-Item $_.FullName ffmpeg}
       - cd ffmpeg
       - git apply --ignore-space-change --ignore-whitespace %APPVEYOR_BUILD_FOLDER%/ffmpeg_plugin/0001-Add-ability-for-ffmpeg-to-run-svt-vp9.patch
```

```suggestion
      - patch --binary -s -N -p1 -i "%APPVEYOR_BUILD_FOLDER%/ffmpeg_plugin/0001-Add-ability-for-ffmpeg-to-run-svt-vp9.patch"
```

I'm not sure if git apply with still patch the files if it's not git